### ISSUE TITLE
UI Improvements of website - home page 

### ISSUE THREAD

**Issue by Ruchip16** [Mentro-Org/Mentro-Community-Blog#2]
Improve the UI of the website i.e [home page](https://community.blog.mentro.tech/)

**Comment by vish-han** [comment]
Can I contribute???

**Comment by LoopThrough-i-j** [comment]
Sure @vish-han

**Comment by Ruchip16** [comment]
> Sure @vish-han.
> Please Check [this](https://github.com/Mentro-Org/Mentro-Community-Blog/blob/main/contributing.md) to get started.
 
yes but Hacktoberfest is yet to start so make sure you [register](https://hacktoberfest.digitalocean.com/) first here.

**Comment by vish-han** [comment]
I'm registered there...
Can u tell ur twitter handle If u ok?
Incase I stuck somewhere....

**Comment by Ruchip16** [comment]
cool, you can go ahead then and comment your doubt here only so that everyone could refer to it

> I'm registered there...
> Can u tell ur twitter handle If u ok?
> Incase I stuck somewhere....

**Comment by SanskritiKhatiyan** [comment]
@Ruchip16 can you tell what kind of ui improvements you need.

**Comment by riyakushwaha** [comment]
I want to work for this issue. Should I start it right away or you would assign it first ans then I can start

**Comment by Ruchip16** [comment]
> @Ruchip16 can you tell what kind of ui improvements you need.

You can comment down your suggestions on the discussions issue and if we feel relevant we would create one & assign that issue to you

**Comment by Ruchip16** [comment]
> I want to work for this issue. Should I start it right away or you would assign it first and then I can start

Yaa go ahead you can submit a PR

**Comment by aumdavda** [comment]
is this issue assigned?

**Comment by Ruchip16** [comment]
> is this issue assigned?

No

**Comment by Ruchip16** [comment]
> is this issue assigned?

I guess you are already being assigned to issue #4

**Comment by aumdavda** [comment]
I have made a PR but it hasn't been accepted since 3 days what shall I do?

**Comment by danish-faisal** [comment]
Hi,
I would like to work on this issue
I find the font unappealing, so would like to change to a more appealing font
And aligning images to the right would make the article overview jumbotron look better

**Comment by LoopThrough-i-j** [comment]
@danish-faisal, please go ahead.
